osx/README.md

```diff
@@ -48,6 +47,30 @@ There are two ways to build the installer, automated or manually.
     It defaults to the `develop` branch at "https://github.com/learningequality/ka-lite/archive/develop.zip".
 
 
+## Changes on installer
+ 
+ * Responsible for setting up environment variable `KALITE_PYTHON`.
+ * Symlink for `Kalite` executable will be done during installation.
+ * Unpack assessment items.
+ * Bundle `KA-Lite.pkg` and `KA-Lite_Uninstall.tool` in `/Applications/KA-Lite` directory.
+
+
+## Uninstaller
+  
+In order to uninstall `KA-Lite.app` and it's dependencies, use `KA-Lite_Uninstall.tool` to uninstall and remove 
+it's dependencies. `KA-Lite.app` and `KA-Lite_Uninstall.tool` are bundled in `/Applications/KA-Lite` folder. 
+In this setup the user can simply locate the uninstaller script and uninstall the `KA-Lite.app`.
+
+During uninstall it removes the following:
+  
+  * `KA-Lite.app` from `/Applications/KA-Lite`.
+  * `org.learningequality.kalite.plist` file in `/Library/LaunchAgents/`.
+  * `/usr/bin/kalite` executable.
```

Comment:
Can you mention "(it checks this location even if it doesn't exist there anymore to remove the previous versions of kalite)".